src/telemetry/knownLoggers.ts

```diff
@@ -120,7 +120,7 @@ export const knownLoggers = (
         path: ENDPOINT,
         options: {
           method: "POST",
-          body: encode("Loggers", apiData),
```

Comment:
My guess is you need to see a specific "loggers" key in this body object vs. just the apiData object?